### DISCUSSION TITLE
Tried to install and run Captions Mod but it doesnt show

### DISCUSSION THREAD

**Discussion by Jn0693** [hzqst/MetaHookSv#460] [Q&A]
Hi

I tried several times to install and run the Captions mod but it doesnt show

I tried to install both One Click Installation and Manual Instalation

One time a windows showed up
![PNG - 0](https://github.com/user-attachments/assets/ba729a15-1d4a-4029-9c4c-d2ca0f4bd974)

Can anyone help me?

**Comment by hzqst** [comment]
You will have to add `VGUI2Extension.dll` in:

`/SteamLibrary/steamapps/common/Sven Co-op/(mod_directory)/metahook/plugins.lst`

or

`/SteamLibrary/steamapps/common/Half-Life/(mod_directory)/metahook/plugins.lst`

**Comment by Jn0693** [comment]
Hi

I tried to install them again and only two work, the Half Life captions and Oposing Force captions.

But in Half Life Uplink maps, Barney's captions dont appear, and in Blue Shift the new captions are not appearing, neither in Blue Shift Training Mode.

Also this message Appears:

![PNG - 1](https://github.com/user-attachments/assets/70116458-c7b6-44db-aeba-f8b6c70d4920)

Im using the dlls in the 1st list that appears when using the bat files:

CaptionMod.dll
VGUI2Extension.dll
BulletPhysics.dll

One last question can it work for the mod Half Life Decay?

**Reply by hzqst** [reply]
1.

Currently, Subtitle directionary files are only available in [Sven Co-op](https://github.com/hzqst/MetaHookSv/blob/main/Build/svencoop/captionmod/dictionary.csv) and [Opposing Force](https://github.com/hzqst/MetaHookSv/blob/main/Build/gearbox/captionmod/dictionary.csv)  (created by community members). However, you can always submit a Pull Request with the subtitle directionary files for other mods, and I will merge them as soon as possible.

2.
As for `BulletPhysics.dll`, it's in WIP state (refactoring) now.

3.
idk

**Comment by Jn0693** [comment]
ok, thank you

is there a section where we can subscribe to see and know if they get done in the future (The Uplink map, Blue Shift and Decay)?
